code/src/layouts/app-footer/AppFooter.tsx

```diff
@@ -0,0 +1,99 @@
+import { FC } from "react";
+import { Link } from "react-router-dom";
+import Box from "@mui/material/Box";
+
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import AppTypography from "@/components/app-typography/AppTypography";
+import {
+  customerSupportFooterItems,
+  locationFooterItems,
+  policyFooterItems,
+  socialsFooterItems
+} from "@/layouts/app-footer/AppFooter.constants";
+
+import "@/layouts/app-footer/AppFooter.scss";
+
+const AppFooter: FC = () => {
```

Comment:
We discussed that we won't use `FC`, just type props directly

Reply:
Thanks for noticing, I forgot to delete it. Fixed